Tasks/GitHubReleaseV0/task.json

```diff
@@ -219,6 +219,30 @@
             "groupName": "changeLogConfiguration",
             "visibleRule": "changeLogCompareToRelease = lastNonDraftReleaseByTag",
             "helpMarkDown": "Specify the regex for release tag. Release matching this tag will be used as base for changelog computation."
+        },
+        
+        {
+            "name": "changeLogType",
+            "type": "radio",
+            "label": "Change log type",
+            "required": true,
+            "defaultValue": "issueBased",
+            "groupName": "changeLogConfiguration",
+            "visibleRule": "addChangeLog = true",
+            "helpMarkDown": "Change log type",
+            "options": {
+                "commitBased": "Commit based",
+                "issueBased": "Issue based"
+            }
+        },
+        {
+            "name": "changeLogLabels",
+            "type": "multiLine",
+            "label": "Labels",
+            "required": true,
+            "visibleRule": "changeLogType = issueBased",
```

Comment:
there should be a default value. We should not have the labels key as it is redundant